python/ray/util/rpdb.py

```diff
@@ -170,12 +170,17 @@ def connect_ray_pdb(host=None, port=None, patch_stdstreams=False, quiet=None):
     return rdb
 
 
-def set_trace(host=None, port=None, patch_stdstreams=False, quiet=None):
+def set_trace():
+    """
+    Interrupt the flow of the program and drop into the Ray debugger.
```

Comment:
```suggestion
    """Interrupt the flow of the program and drop into the Ray debugger.
```
the summary line needs to be on the first line (for rendering purposes).